docs/changes/table/11660.other.rst

```diff
@@ -0,0 +1,2 @@
+For mixins, ``Table`` shows the object's class' qualitative name, not "object",
```

Comment:
I would call this an API change since it introduces a substantive change in the output of certain table methods. Thus change the name to `11660.api.rst`. However, I don't think this will be a highly impacting change for most users since those print/format methods are largely used for interactive work. However, like you have seen here in astropy there might be a number of doctests or unit tests that start failing as a result.

Reply:
Good point. Changing the designation.